internal/compiler-interface/src/main/datatype/incremental.json

```diff
@@ -146,6 +146,14 @@
           "name": "externalHooks",
           "type": "xsbti.compile.ExternalHooks",
           "doc": "External hooks that allows clients e.g. IDEs to interacts with compilation internals"
+        },
+        {
+          "name": "fastExtDependenciesLookup",
```

Comment:
I think we need a better name than "fast", since you might come up with even faster ideas, and then we'd have to call it "superFast".